package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4235,6 +4239,9 @@ def __init__(self, *args, **kwargs):
         .. versionchanged:: 0.11.0
            Added the *is_anchor* and *anchor_name* keywords for finer behavior
            control when unpickling instances of :class:`MDAnalysis.core.AtomGroup.AtomGroup`.
+        .. versionchanged:: 0.15.0
+           *permissive* set to ``True`` and set to ``False`` both yield permissive
```

Comment:
The "permissive" flag is not used anymore (and effectively defaults to `True`); it will be completely removed in 0.16.0.